src/keyszer/transform.py

```diff
@@ -216,6 +216,31 @@ def dump_diagnostics():
     print("")
 
 
+# ─── COMBO CONTEXT LOGGING ────────────────────────────────────────────────────────
+
+
+def log_combo_context(combo, ctx, keymap, _active_keymaps):
+    """Log context around usage of combo"""
+    import textwrap
+
+    debug("")
+    debug(f"WM_CLS: '{ctx.wm_class}' | WM_NME: '{ctx.wm_name}'")
+    debug(f"DVN: '{ctx.device_name}' | CLK: '{ctx.capslock_on}' | NLK: '{ctx.numlock_on}'")
```

Comment:
```suggestion
    debug(f"WM_CLASS: '{ctx.wm_class}' | WM_NAME: '{ctx.wm_name}'")
    debug(f"DEVICE: '{ctx.device_name}' | CAPS_LOCK: '{ctx.capslock_on}' | NUM_LOCK: '{ctx.numlock_on}'")
```

Not worth the trouble vs just making it readable.

Reply:
> Not worth the trouble vs just making it readable. 

That’s funny. I was literally just pondering doing that.